main.js

```diff
@@ -0,0 +1,98 @@
+var button = document.getElementById('submit');
+var commentsBlock = document.getElementsByClassName('comments-block')[0];
+
+button.addEventListener('click', function () {
```

Comment:
Why are you not using jQuery?

How could you write this so that not everything is inside of this one event listener?